File-Formats/PDF/Working-with-Security.md

```diff
@@ -1336,3 +1336,90 @@ Returns user password</td><td>
 Returns owner Password (owner password is same as the user password; it allows full permission to users).</td></tr>
 </tbody>
 </table>
+
+## Troubleshooting
+
+ If the document is protected with owner password only, it seems that it can be open without entering password. In that case, can the Syncfusion library remove/change restrictions? And how do you open a document using owner password if it?s different from user password?
```

Comment:
If the document is protected with the owner's password only, it can be opened without entering a password. In that case, can the Syncfusion library remove or change restrictions? And how do you open a document using the owner's password if it's different from the user's password?